components/ep/layout/SurroundDocCard.vue

```diff
@@ -31,7 +31,9 @@ defineProps<{
 
 <template>
   <NuxtLink
-    :to="wTrailingSlash(doc._path)"
+    :to="
+      typeof doc._path === 'string' ? withTrailingSlash(doc._path) : doc._path
+    "
```

Comment:
There can be a shorten expression

> If doc._path is `undefined` provide `''` instead.

```suggestion
    :to="withTrailingSlash(doc._path ?? '')"
```